src/styles/utils/variables.scss

```diff
@@ -0,0 +1,5 @@
+$base-size: 100px;
+$base-gap: 1px;
+$day-color: #eee;
+$hover-color: #ffbfcb;
+$day-names: 'mon' 1, 'tue' 2, 'wed' 3, 'thu' 4, 'fri' 5, 'sat' 6, 'sun' 7;
```

Comment:
Ensure that the usage of the `$day-names` map aligns with the Code Style Rules. If this map is intended for styling purposes, consider using more descriptive names for the keys or adding a comment to explain the usage of this map.